<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Robert B. Scharpf Home  by rscharpf</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/pygment_trac.css">
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>
  <body>
    <div class="wrapper">
      <header>
        <h1>Robert B. Scharpf</h1>
	<img src="rs_small.jpg">
<!--        <p>Research Page</p> -->
        <p class="view"><a href="https://github.com/rscharpf">View My GitHub Profile</a></p>
        <p class="view"><a href="talks.html">Talks</a></p>
<!--        <p class="view"><a href="Teaching/index.html">Teaching</a></p> -->
        <p class="view"><a href="teaching.html">Teaching</a></p>
        <p class="view"><a href="http://scholar.google.com/citations?user=Zh6R2YYAAAAJ">Google citations</a></p>
        <p class="view"><a href="cv.pdf">CV [pdf]</a></p>
        <p class="view"><a href="https://www.rits.onc.jhmi.edu/DBB/members/?members=Research&member=rscharp1">Official website</a></p>
        <p class="view"><a>Contact: </a><BR>
	    550 N. Broadway, Suite 1103 <BR>
	    Baltimore, MD 21205-2013<BR>
	    Phone: 443 287 9408<BR>
	    FAX: 410 955 0859<BR>
	    email: <a href="mailto:rscharpf@jhsph.edu">rscharpf_at_jhsph_dot_edu</a><BR>
      </header>
      <section>
        <h3>
<a name="research-overview" class="anchor" href="#research-overview"><span class="octicon octicon-link"></span></a>Research Overview</h3>

<p>I develop statistical methods and software for the analysis of
high-throughput genomic data, including methods for estimation of
cross-study differential gene expression from mRNA microarrays and
detection of DNA structural variants from array- and sequence-based
platforms.  I also develop methods and software for preprocessing
array and sequence platforms to remove batch effects and other
technical sources of variation.  Software is available from
Bioconductor, with developmental versions available from github.</p>

<h3>
<a name="bioconductor-packages" class="anchor" href="#bioconductor-packages"><span class="octicon octicon-link"></span></a>Bioconductor Packages</h3>

<ul>
<li>oligoClasses: infrastructure for crlmm and oligo packages </li>
<li>crlmm:  Methods for calling genotypes and copy number for Affymetrix and Illumina genotyping platforms</li>
<li>SNPchip: A few plotting utilities for log R ratios / BAFs; chromosome idiograms</li>
<li>VanillaICE: A hidden Markov model for inferring copy number alterations from genotyping arrays</li>
<li>XDE: A Bayesian Hierarchical model for cross-study differential gene expression</li>
<li>MinimumDistance: Methods for calling de novo copy number variants in case-parent trios</li>
<li>ArrayTV: Software for removing genomic waves from marker-level estimates of copy number such as log R ratios derived Affymetrix or Illumina genotyping arrays</li>
</ul><p>The above packages can be installed at the R prompt by</p>

<pre><code>> source("http://www.bioconductor.org/biocLite.R")
> pkgs &lt;- c("oligoClasses, "crlmm", "XDE", "MinimumDistance", "ArrayTV", "SNPchip", "VanillaICE")
> biocLite(pkgs)
</code></pre>

<h3>
<a name="github-packages" class="anchor" href="#github-packages"><span class="octicon octicon-link"></span></a>GitHub Packages</h3>

<p>The developmental versions of these packages (use at your own risk) are available on github can be installed by</p>

<pre><code>> install.packages("devtools")
> library(devtools)
> install_github("MinimumDistance")
</code></pre>

<p>Note these packages are tested against the developmental version of R.</p>
      </section>
      <footer>
        <p><small>Hosted on GitHub Pages &mdash; Theme by <a href="https://github.com/orderedlist">orderedlist</a></small></p>
      </footer>
    </div>
    <script src="javascripts/scale.fix.js"></script>

  </body>
</html>
